### DISCUSSION TITLE
Changelog

### DISCUSSION THREAD

**Discussion by cdmoro** [cdmoro/bootstrap-vue-3#185] [Ideas]
We are growing up and I think it's time to create and maintain a changelog, do you know a tool or something to implement it? Or is there a way to normalize commits, use "Conventional commits" par example?

**Comment by hi-reeve** [comment]
maybe [conventional-changelog](https://github.com/conventional-changelog/conventional-changelog) ?

**Reply by et84121** [reply]
I think this would be a great option.
